package.json

```diff
@@ -110,7 +110,7 @@
 		"@hashicorp/react-code-block": "^6.6.0",
 		"@hashicorp/react-consent-manager": "^9.2.0",
 		"@hashicorp/react-content": "^8.3.0",
-		"@hashicorp/react-design-system-components": "^0.0.0-nightly.0c364f3",
+		"@hashicorp/react-design-system-components": "^0.0.0-nightly.c797020",
 		"@hashicorp/react-design-system-tokens": "^0.0.0-nightly.8920560",
```

Comment:
@zchsh can you bump `react-design-system-tokens` to the same version? There's nothing new, but I want to make sure we keep the versions aligned.

Reply:
Great call - done! ✅